app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -74,10 +74,9 @@ export default class DiscoverySidebar extends React.Component {
           // avg_reads_per_sample: 0,
         },
         metadata: {
-          // TODO (gdingle): these freq counts per project, not per sample
```

Comment:
is this not a problem for the UI? are we expecting to show stats per project in the project view, not weighted per sample?

Reply:
I thought it would be per project, when in the projects page.